.github/dependabot.yml

```diff
@@ -12,3 +7,30 @@ updates:
       interval: "daily"
     open-pull-requests-limit: 5
     target-branch: "master"
+    groups:
+      jakarta:
+        patterns:
+        - "jakarta.*"
+        update-types:
+        - "patch"
+      microprofile:
+        patterns:
+        - "org.eclipse.microprofile*"
+        update-types:
+        - "patch"
+    ignore:
+      - dependency-name: "jakarta.*"
+        update-types: ["version-update:semver-major"]
+      - dependency-name: "jakarta.*"
+        update-types: ["version-update:semver-minor"]
+      - dependency-name: org.eclipse.microprofile*"
+        update-types: ["version-update:semver-major"]
+      - dependency-name: "org.eclipse.microprofile*"
+        update-types: ["version-update:semver-minor"]
```

Comment:
Do we need separate entries for ignoring major and minor?
It's an array isn't it?
Can we not have:
```
    ignore:
      - dependency-name: "jakarta.*"
        update-types: ["version-update:semver-major", "version-update:semver-minor"]
      - dependency-name: org.eclipse.microprofile*"
        update-types: ["version-update:semver-major", "version-update:semver-major"]
```

Reply:
updated in single entries

Reply:
I created a new PR. Iam not able to push to this branch as its not from my Forked repo.